### DISCUSSION TITLE
Change default mouse keybind for global back form M4 to M3

### DISCUSSION THREAD

**Discussion by ArchUsr64** [ppy/osu#30771] [Ideas] [answered]
The game currently uses M4 for the global back operation and M3 is only used for pause/resume during gameplay. Since M3 aka the middle mouse button is available on almost every computer mice whereas M4 is usually only found on gaming mice. I suggest changing the default keybind for global back operation to M3 as it would improve mouse-only navigation for non gaming mice users.

## Current Defaults:
![image](https://github.com/user-attachments/assets/b9456ab6-4626-4f58-a93e-118555027406)
## Suggestion:
![image](https://github.com/user-attachments/assets/78e6d552-a177-41b6-b866-bcfd41d11cd1)

**Comment by peppy** [comment, marked as answer]
Middle mouse has never been a back button in any application I've used. M4 is the established standard.
